tests/mtd_flashpage/main.c

```diff
@@ -217,6 +218,80 @@ static void test_mtd_write_read_page(void)
 #endif
 }
 
+#if CONFIG_SLOT_AUX_LEN
+static bool mem_is_all_set(const uint8_t *buf, uint8_t c, size_t n)
+{
+    for (const uint8_t *end = buf + n; buf != end; ++buf) {
+        if (*buf != c) {
+            return false;
+        }
+    }
+
+    return true;
+}
+
+static void test_mtd_aux_slot(void)
+{
+    mtd_dev_t *dev = mtd_aux;
```

Comment:
There is already `static mtd_dev_t *dev = &_dev.base;` above. I know, local scope wins but lets not give it the same name